fluentlenium-core/src/main/java/org/fluentlenium/core/Fluent.java

```diff
@@ -360,6 +360,32 @@ public Fluent goTo(String url) {
         return this;
     }
 
+    /**
+     * Open the url page in a new tab
+     *
+     * @param url
+     */
+    public Fluent goToInNewTab(String url) {
+        if (url == null) {
+            throw new IllegalArgumentException("Url is mandatory");
+        }
+
+        if (baseUrl != null) {
+            URI uri = URI.create(url);
+            if (!uri.isAbsolute()) {
+                url = baseUrl + url;
+            }
+        }
+
+        Set<String> initialTabs = getDriver().getWindowHandles();
+        executeScript("window.open('" + url + "', '_blank');");
```

Comment:
I've just tested on Firefox. The option mentioned on SO seems a bad idea because it is OS dependent.